dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ConjureBodySerDe.java

```diff
@@ -67,31 +75,49 @@ final class ConjureBodySerDe implements BodySerDe {
      */
     ConjureBodySerDe(
             List<WeightedEncoding> rawEncodings,
-            ErrorDecoder errorDecoder,
+            ErrorDecoder _errorDecoder,
```

Comment:
We don't need to pass this in. `DefaultConjureRuntime` is the public API where this ctor is created, and it passes in a hardcoded `ErrorDecoder.INSTANCE`.